docs/content/developers/design-keuzes.md

```diff
@@ -121,14 +121,12 @@ De URLs in dit voorbeeld zijn uiteraard fictief.
   ```
 
 
-## Gerelateerde objecten zonder eigen resource / Groepsattributen
+## Groepsattributen
 
-Indien een (hoofd)object een gerelateerd object (of lijst van objecten) heeft,
-dat **geen** eigen resource URL nodig heeft, dan wordt het (child)object inline
-(binnen het hoofdobject) ontsloten. Deze child objecten zijn binnen RGBZ
-ook wel gedefinieerd als *groepsattributen*. Typische voorbeelden zijn
-(zaak-)kenmerken of (zaak-)eigenschappen waarin `Zaak` het hoofdobject betreft,
-en `Kenmerk` en `Eigenschap` de child objecten.
+Indien een object een groepattribuutsoort heeft (een groep van bij elkaar behorende attribuutsoorten), al dan niet herhalend d.w.z. kardinaliteit 1 of meer (een 'lijst' van waarden van de attributen van het groepattribuut), dan wordt het groepattribuut inline
+(binnen het object) ontsloten. Het krijgt dus geen eigen resource, het heeft immers geen zelfstandig bestaantsrecht. Het heeft alleen bestaansrecht als eigenschap van het object, net zoals attribuutsoorten. 
```

Comment:
> Het heeft alleen bestaansrecht als eigenschap van het object

Dit verandert twee voorbeelden (eigenschappen en kenmerken) in iets dat 'een eigenschap' is. Is het niet beter om de voorbeelden te noemen zodat de lezer zich er concreet wat bij kan voorstellen?

Reply:
Met de regel hieronder, denk ik eigenlijk dat deze lijn zelfs helemaal weggelaten kan worden. Het gaat m.i. te veel in op de semantiek van RGBZ, en als developer die met REST/json bezig is, heb ik hier eigenlijk niet echt wat aan.

Reply:
Wellicht wil je als developer alleen de ontwerpkeuze weten. Ik wil ook graag het waarom daarvan weten. Zodat later te achterhalen is wat de reden van de keuze is. Daar kun je namelijk oer discussieren, niet over de keuze zelf. Wellicht voor elk ontwerpkeuze-onderwerp eerste de ontwerpkeuze vermelden en onder een apart kopje de reden (motiverin) daarvan?

Reply:
Ik weet het niet goed - @joeribekker, wat denk jij?

Reply:
typo in "bestaantsrecht" :)

Typisch gevalletje RGBZ --> REST API met beide hun eigen steekwoorden.
Als rationale lijkt mij het prima dit er in te zetten. De design keuzes zijn eigenlijk wat nu "verstuffing" heet, met het verschil dat het de design keuzes (en visie, en uitgangspunten) nu ook een wijziging van het RGBZ kunnen veroorzaken.

De voorbeelden gaan er niet uit toch?

Reply:
Overeenkomstig de laatste comment niets aangepast, alleen een typefout hersteld.